launchers/ids-connector/src/main/java/org/eclipse/dataspaceconnector/runtime/EdcRuntime.java

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
These EdcRuntime files are no longer needed. Instead in the build.gradle.kts file you should replace
`
application {
    @Suppress("DEPRECATION")
    mainClassName = "org.eclipse.dataspaceconnector.runtime.ConnectorRuntime"
}
`

by

`
application {
    @Suppress("DEPRECATION")
    mainClassName = "org.eclipse.dataspaceconnector.system.runtime.BaseRuntime"
}
`